common/fe-ste-models.go

```diff
@@ -1609,3 +1611,33 @@ func (rpt RehydratePriorityType) ToRehydratePriorityType() azblob.RehydratePrior
 		return azblob.RehydratePriorityStandard
 	}
 }
+
+////////////////////////////////////////////////////////////////////////////////
+type SymlinkHandlingType uint8 // SymlinkHandlingType is only utilized internally to avoid having to carry around two contradictory flags. Thus, it doesn't have a parse method.
+
+// for reviewers: This is different than we usually implement enums, but it's something I've found to be more pleasant in personal projects, especially for bitflags. Should we change the pattern to match this in the future?
+
+type eSymlinkHandlingType uint8
+
+var ESymlinkHandlingType = eSymlinkHandlingType(0)
+
+func (eSymlinkHandlingType) None() SymlinkHandlingType     { return SymlinkHandlingType(0) }
```

Comment:
Shall we rename None() to Skip() which is more explicit?

Reply:
Makes sense to me.